tests/test-cases.js

```diff
@@ -1564,6 +1564,72 @@ var CUSTOM_TESTS = [
 			});
 		}
 	},
+	{
+		description: "Chunk functions can pause parsing",
+		expected: [
+			[['A', 'b', 'c']]
+		],
+		run: function(callback) {
+			var updates = [];
+			Papa.parse('A,b,c\nd,E,f\nG,h,i', {
+				chunkSize: 10,
+				chunk: function(response, handle) {
+					updates.push(response.data);
+					handle.pause();
+					callback(updates);
+				},
+				complete: function() {
+					callback(new Error('incorrect complete callback'));
+				}
+			});
+		}
+	},
+	{
+		description: "Chunk functions can resume parsing",
+		expected: [
+			[['A', 'b', 'c']],
+			[['d', 'E', 'f'], ['G', 'h', 'i']]
+		],
+		run: function(callback) {
+			var updates = [];
+			var handle = null;
+			var first = true;
+			Papa.parse('A,b,c\nd,E,f\nG,h,i', {
+				chunkSize: 10,
+				chunk: function(response, h) {
+					updates.push(response.data);
+					if (!first) return;
+					handle = h;
+					handle.pause();
+					first = false;
+				},
+				complete: function() {
+					callback(updates);
+				}
+			});
+			setTimeout(function() {
+				handle.resume();
+			}, 500);
+		}
+	},
+	{
+		description: "Chunk functions can abort parsing",
+		expected: [
+			[['A', 'b', 'c'], ['d', 'E', 'f'], ['G', 'h', 'i']]
```

Comment:
Shouldn't be better to expect only a partial result?

Otherwise if the handle does not abort it will return the same results and it will not fail when it should

Reply:
Done. 

As a follow up, using something like Sinon for spies makes sense to verify `abort` was called.